.gitignore

```diff
@@ -14,9 +14,9 @@ docs/_build
 env
 venv/
 
-examples/checkpoint*
+examples/**/checkpoint*
+examples/**/checkpoints/
```

Comment:
doesn't `examples/**/checkpoint*` match `examples/**/checkpoints/?

Reply:
Yes, it should, good point.  That was a holdover from the previous version.  Will fix that.